pageserver/src/layered_repository/layer_map.rs

```diff
@@ -292,11 +288,9 @@ impl LayerMap {
 
     /// Return the oldest in-memory layer.
     pub fn peek_oldest_open(&self) -> Option<Arc<InMemoryLayer>> {
-        if let Some(opensegentry) = self.open_segs.peek() {
-            Some(Arc::clone(&opensegentry.layer))
-        } else {
-            None
-        }
+        self.open_segs
```

Comment:
same here, the map() doesn't look as readable to me.

Reply:
This change become irrelevant with rebase to current main